### PR TITLE
Resolve Coupling between `scanid` and `instantsearch` packages

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.clubhouse.io/simpledotorg/story/3639/resolve-coupling-between-scanid-and-instantsearch-packages

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/instantsearch/InstantSearchModel.kt [hunk 1]
```diff
@@ -15,7 +15,7 @@ data class InstantSearchModel(
     val additionalIdentifier: Identifier?,
     val instantSearchProgressState: InstantSearchProgressState?,
     val scannedQrCodeSheetAlreadyOpened: Boolean,
-    val patientPrefillInfo: PatientPrefillInfo?
+    val indiaNHIDInfo: IndiaNHIDInfo?
```
[the remark this answers is not in the document]
Comment (janhavisinghh) [reply app/src/main/java/org/simple/clinic/instantsearch/InstantSearchModel.kt]: How does `indiaNHIDPrefillInfo` sound?

**Comment by msasikanth** [comment]
It has to be country agnostic right? Since we can support other payloads in
future (we will definitely). Why not follow the second suggestion Vinay
mentioned, extracting out the patient prefill info object into a separate
common package, kinda like how we have `OngoingNewPatientEntry` in
`patient` package. That way we don't have to worry about creating 2
different objects.

**Comment by janhavisinghh** [comment]
> different objects.

Umm, I think we can put `PatientPrefillInfo` in `patient` package as well. As it is related to the `patient`. Wdyt?

**Review by msasikanth — COMMENTED** [review]
> > different objects.
> 
> Umm, I think we can put `PatientPrefillInfo` in `patient` package as well. As it is related to the `patient`. Wdyt? @msasikanth

Yes, that's what I said in the above comment 😅

**Review by msasikanth — APPROVED** [review]

**Review by msasikanth — APPROVED** [review]

**Pull request merged**